### DISCUSSION TITLE
don´t see it, where is it the app

### DISCUSSION THREAD

**Discussion by RodrigoB2023** [Parallels/parallels-vscode-extension#145] [Q&A]
Hello ,  Please your kindly help. I had installed the Parallels vscode extension but  I do not see it. 
I  will like to have the user manual with  instructions on how to use this extension, I do not see where can I find the app where to set all this wonderful settings. 

Thanks in advance,

Rodrigo

**Comment by cjlapao** [comment]
@RodrigoB2023 

The extension will show on the side of VSCode like so.

![image](https://github.com/user-attachments/assets/d47b3d70-72ec-4afd-9293-44f37bfb8b83)
there is no user manual for this as it is an extension
